firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/FunctionCallingConfig.kt

```diff
@@ -17,13 +17,7 @@
 package com.google.firebase.vertexai.type
 
 /**
- * Contains configuration for function calling from the model. This can be used to force function
- * calling predictions or disable them.
- *
- * @param mode The function calling mode of the model
- * @param allowedFunctionNames Function names to call. Only set when the [Mode.ANY]. Function names
- * should match [FunctionDeclaration.name]. With [Mode.ANY], model will predict a function call from
- * the set of function names provided.
+ * The configuration that specifies the function calling behavior.
```

Comment:
We should provide more info here

Reply:
It's meant to be short because the actual documentation is in the factory methods that describe what behaviour each instance represents.

Reply:
Maybe provide a `@see` block for the factory methods?